atlp-rwanda/commanders-rn-medica
# Feature: doctor appointment payment screens

#### What does this PR do?

This PR includes designs of payment screens for the doctor appointment screens

#### Description of Task to be completed?

- Design select payment methods screen
- Design add new card screen
- Design summary screen
- Design success & failed state screens

#### How should this be manually tested?

Navigate to the home page and scroll down. Click on one doctor to view their details. Click next button until you get to the payments screen. Click add new card to see the screen for adding a new card. Go back and click Next to see summary of the payments and click next to input your pin. After clicking next you should see a success modal.

#### Any background context you want to provide?

N/A

#### What are the relevant pivotal tracker/Trello stories?

N/A

#### Screenshots (if appropriate)

<img width="180" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/fc91e2c4-97b6-4e0b-a185-56f5931766d8" />
<img width="180" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/11c3e4b1-8408-4e8e-9802-374cc28b15ed" />
<img width="180" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/3d26e03f-3fd3-4ff5-9977-d4c6e6ec08d3" />
<img width="180" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/02aab846-8c91-4a4f-a2eb-bade2edf906d" />
<img width="180" src="https://github.com/atlp-rwanda/commanders-rn-medica/assets/51389166/92a8c3b4-09d6-4800-8ccd-e0b173ee2e63" />

#### Questions:

N/A
